docs/source/notebooks/tutorial.ipynb

```diff
@@ -834,52 +974,26 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Loading data\n",
-    "\n",
-    "The best part about `intake-esm` is that it enables loading data directly into an [xarray.Dataset](http://xarray.pydata.org/en/stable/api.html#dataset) via `to_dataset_dict()` method.\n",
-    "\n",
-    "\n",
-    "\n",
-    "Note that \n",
-    "- the data could be stored in the cloud or on a local filesystem, \n",
-    "- the data could be stored in [zarr](https://zarr.readthedocs.io/en/stable/) format, [netCDF](https://www.unidata.ucar.edu/software/netcdf/) format. \n",
+    "## Loading datasets\n",
     "\n",
-    "**And all of this is opaque to the user!**\n",
+    "As pointed earlier, intake ESM contains special logic to figure out how to concatenate and merge the individual results of your query into larger, more high-level aggregated [xarray datasets](http://xarray.pydata.org/en/stable/api.html#dataset).\n",
```

Comment:
"As pointed out earlier, intake-esm contains logic to merge the query results into higher level [xarray datasets]"

Reply:
Although my background may bias me, I think most users will already have some intuition for the benefits of xarray objects over individual files.